src/service/extension-location.js

```diff
@@ -59,32 +47,10 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev,
  * @param {!Location} location The window's location
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
- * @param {boolean=} isTest
  * @return {string}
  */
-export function calculateEntryPointScriptUrl(location, entryPoint, isLocalDev,
-    isTest) {
-  const base = calculateScriptBaseUrl(location, isLocalDev, isTest);
-  const serveMax = isLocalDev && isMax(location);
```

Comment:
good news is with this PR, gulp server will use min files with `--compiled` flag.  we don't need to deal with `isUsingCompiledJs` anymore.